src/Components/NavigationBar/NavigationBar.tsx

```diff
@@ -1,18 +1,64 @@
 import { NavLink } from 'solid-app-router';
 import type { Component } from 'solid-js'
-import { navSignal } from '../../Store/navigation-bar-store'
+import { docsNavSignal, navSignal } from '../../Store/navigation-bar-store'
+import { createSignal } from 'solid-js';
+import { Show } from 'solid-js'
 
 export const NavigationBar: Component = () => {
-    const [navs, _] = navSignal;
-
+    const [showDocumentationItems, setShowDocumentationItems] = createSignal(false);
+    const [showMobileNav, setShowMobileNav] = createSignal(false);
```

Comment:
I also wonder about createSignals inside components. It is similar to useState, however as we can se in navigation-bar-store.ts we can also place them outside components (which is super cool). So I wonder that when you place crateSignal inside `NavigationBar` is it because you treat them as local states? Or could they be outside component?

Reply:
Yes I used them inside this file because they are all local state and we don't need them from outside!